packages/platform/atoms/calendar-view/wrappers/CalendarViewPlatformWrapper.tsx

```diff
@@ -13,21 +39,115 @@ export type CalendarViewPlatformWrapperAtomPropsForTeam = {
   eventSlug: string;
 };
 
-type EventTypeCalendarViewProps = { isEventTypeView: true } & (
-  | (CalendarViewPlatformWrapperAtomPropsForIndividual & { teamId?: number })
-  | (CalendarViewPlatformWrapperAtomPropsForTeam & { username?: string | string[] })
-);
+const CalendarViewPlatformWrapperComponent = (
+  props:
+    | (CalendarViewPlatformWrapperAtomPropsForIndividual & { teamId?: number })
+    | (CalendarViewPlatformWrapperAtomPropsForTeam & { username?: string | string[] })
+) => {
+  const isTeamEvent = !!props.teamId;
+  const teamId: number | undefined = props.teamId ? props.teamId : undefined;
+  const username = useMemo(() => {
```

Comment:
This is a lot of new code added thats basically copy from booker platform wrapper. If this code here needs `orgSlug`, can't we pass org slug from `const setOrg = useBookerStoreContext((state) => state.setOrg)` ? state but just access the set org in state?